app/start_handlers.py

```diff
@@ -222,6 +231,28 @@ async def get(self, request):
         }
 
 
+@start_routes.view('/ni/start/code-for-ce-manager/')
+class StartNICE4Code(StartCommon):
+    @aiohttp_jinja2.template('start-code-for-ni-ce-manager.html')
+    async def get(self, request):
+        self.setup_request(request)
+        display_region = 'ni'
+        self.log_entry(request, display_region + '/start/code-for-ce-manager')
+
+        locale = 'en'
+        page_title = 'This access code is not part of the census for England and Wales'
```

Comment:
Does this need to be translated into Welsh when it get displayed in http://localhost:9092/cy/start/code-for-northern-ireland/?

Reply:
As discussed, this can only be for NI, so is only in english. The route for en/cy does include translation markers.